syntax = "proto2";
package mmo.proto_client;

message ModelDataEntry {
	required uint32 id = 1;
	required string name = 2;
	optional uint32 flags = 3;
	required string filename = 4;
	map<uint32, uint32> customizationProperties = 5;
}

message ModelDatas {
	repeated ModelDataEntry entry = 1;
}
